src/easylink/configuration.py

```diff
@@ -108,31 +160,14 @@ def spark_resources(self) -> Dict[str, Any]:
             **{k: v for k, v in self.spark.items() if k != "workers"},
         }
 
-    def get_implementation_specific_configuration(self, step_name: str) -> Dict[str, Any]:
-        """Extracts and formats an implementation-specific configuration from the pipeline config"""
-
-        def _stringify_keys_values(config: Any) -> Dict[str, Any]:
-            # Singularity requires env variables be strings
-            if isinstance(config, Dict):
-                return {
-                    str(key): _stringify_keys_values(value) for key, value in config.items()
-                }
-            else:
-                # The last step of the recursion is not a dict but the leaf node's value
-                return str(config)
-
-        config = self.pipeline["steps"][step_name]["implementation"].get("configuration", {})
-        return _stringify_keys_values(config)
-
     def get_implementation_name(self, step_name: str) -> str:
         return self.pipeline["steps"][step_name]["implementation"]["name"]
 
     #################
     # Setup Methods #
     #################
-
     @staticmethod
-    def _determine_if_spark_is_required(pipeline: Dict[str, Any]) -> bool:
+    def _determine_if_spark_is_required(pipeline) -> bool:
```

Comment:
We're going to move this repo to fully typed soon

Reply:
whoops, this change was unintended. I was messing around with making it not static